src/ansys/geometry/core/designer/designpoint.py

```diff
@@ -22,8 +22,10 @@
 """Module for creating and managing design points."""
 
 from beartype.typing import TYPE_CHECKING, Union
+import pyvista as pv
```

Comment:
Perform the import inside the _to_polydata method if possible @AlejandroFernandezLuces - that will avoid loading PyVista unless strictly necessary for plotting